packages/api/src/APIProvider.tsx

```diff
@@ -20,11 +23,28 @@ const getSharedQueryClientContext = (): QueryClient => {
 
 const queryClient = getSharedQueryClientContext();
 
-const APIProvider = ({ children }: PropsWithChildren<unknown>) => (
-    <QueryClientProvider client={queryClient}>
-        {children}
-        <ReactQueryDevtools />
-    </QueryClientProvider>
-);
+type TProps = {
+    /** If set to true, the APIProvider will instantiate it's own socket connection. */
+    standalone?: boolean;
+};
+
+const APIProvider = ({ children, standalone = false }: PropsWithChildren<TProps>) => {
+    const deriv_api = useRef(new DerivAPIBasic({ endpoint: 'qa29.deriv.dev', app_id: 9999, lang: 'ES' }));
```

Comment:
it is intended to hardcode the params 👀

Reply:
@farhan-nurzi-deriv Of course not man 😁 Just testing for now, Once we are sure it works and doesn't break anything then we can fix stuff 🙇🏻